packages/gatsby-theme-docs/src/templates/homepage.js

```diff
@@ -66,6 +58,21 @@ HomepageTemplate.propTypes = {
 };
 export default HomepageTemplate;
 
+// eslint-disable-next-line react/prop-types
```

Comment:
It looks like there is no way to define PropTypes for the Head function. So I needed to disable eslint for the lines where we define the props for the SEO component.